tools/harness-simulation/posix/launch_testbed.py

```diff
@@ -131,53 +128,41 @@ def main():
 
     # Parse arguments
     parser = argparse.ArgumentParser()
-
-    # Determine the interface
-    parser.add_argument('-i',
-                        '--interface',
-                        dest='ifname',
+    parser.add_argument('-c',
+                        '--config',
+                        dest='config',
                         type=str,
                         required=True,
-                        help='the interface used for discovery')
-
-    # Determine the number of OpenThread FTD simulations to be "detected" and then started
-    parser.add_argument('--ot',
-                        dest='ot_num',
-                        type=int,
-                        required=False,
-                        default=0,
-                        help=f'the number of OpenThread FTD simulations')
-
-    # Determine the number of OpenThread BR simulations to be initiated and then detected
-    parser.add_argument('--otbr',
-                        dest='otbr_num',
-                        type=int,
-                        required=False,
-                        default=0,
-                        help=f'the number of OpenThread BR simulations')
-
-    # Determine the number of sniffer simulations to be started and then detected
-    parser.add_argument('--sniffer',
-                        dest='sniffer_num',
-                        type=int,
-                        required=False,
-                        default=0,
-                        help=f'the number of sniffer simulations, no more than {MAX_SNIFFER_NUM}')
-
+                        help='the path of the configuration JSON file')
     args = parser.parse_args()
+    with open(args.config, 'r') as f:
```

Comment:
```suggestion
    with open(args.config, 'rt') as f:
```
nit

Reply:
I wonder why it's necessary to specify `t`? I thought it's by default enabled.

Reply:
Both `r` and `t` flags are enabled by default. For consistency, I think either `open(filename, 'rt')` or `open(filename)` is acceptable.

Reply:
't' is indeed default. But still better to be explicit.